en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
+### Video Selection
 
-Choosing multiple videos is often the best approach for the exploratory stages of research, because the YouTube API may not return data for every video searched, even if comment data for that video exists. Wordfish modeling is the most useful, and the most externally valid (meaningful outside the scope of a specific dataset), when the data being modeled is focused on a single topic, but encompasses a variety of viewpoints.
+The most direct way to select videos for research is to visit the [YouTube](https://www.youtube.com/) site, and capture a list of video IDs: these are the set of alphanumeric characters that appear in the video's URL, immediately after `watch?v=`. See for example the video ID circled in red in the illustration below: `24xsqyMcpRg`. These IDs are constant and do not change over time. 
 
-For politically salient topics, an ideal dataset will include an equal number of videos from creators representing both the opposing political perspectives, and should have a substantial number of comments (~2000+) to minimize the skew that outlier comments can introduce. Finally, the total comments from each viewpoint should be balanced. In this case, we made sure that comments from the political ‘left’ and ‘right’ were similar in number. 
+{% include figure.html filename="en-or-text-mining-youtube-comments-02.png" alt="Screenshot of YouTube video with video ID in browser link circled in red" caption="Figure 2. Screenshot of YouTube video with video ID in browser link circled in red" %}
 
-## Querying YouTube's API
-We recommend using YouTube Data Tools to query YouTube’s API. 
+For this lesson, we gathered comment data by searching YouTube for 'black lives matter george floyd'. We selected a total of six videos from politically polarized news sources (as ranked by [allsides.com](allsides.com)), from the left-leaning New York Times, Vox, and NBC News, and the right-leaning Daily Mail, Fox News, and the Daily Wire. Choosing multiple videos is often the best approach for the exploratory stages of research, because while YouTube makes available a wide range of metadata about each video (number of likes, title, description, tags and more), the YouTube API may not return comment data for every video searched. For politically salient topics, an ideal dataset will include an equal number of videos from creators representing both the opposing political perspectives, and should have a substantial number of comments (~2000+) to minimize the skew that outlier comments can introduce. Finally, the total comments from each viewpoint should be balanced. In this case, we made sure that comments from the political left and right were similar in number. 

```

Comment:
I was a little confused by the last sentence ('Finally, [...]'). I'm not sure how you are able to 'make sure that comments from the political left and right were similar in number'. Do you mean 'comments posted under videos from the political right were similar in number to those posted under videos from the political left'?

Reply:
That's a good point, here's another way to phrase the last few sentences, which are slightly misleading. The following should be a new paragraph:

"For politically salient topics, an ideal dataset will include a similar number of videos from creators representing opposing political perspectives. The video sources representing each side of the political spectrum should contribute a similar number of comments to the total corpus. Finally, the dataset should have a substantial number of comments (~2000+) to minimize the skew that outlier comments can introduce."